man/utop.1

```diff
@@ -98,11 +98,115 @@ with options available for
 .BR ocaml (1).
 
 A commonly used option is
-\fB--require\fI package\fR
+\fB-require\fI package\fR
 to load \fIpackage\fR into the execution environment. It is equivalent
 to using \fb#require\fR from inside
 .BR utop (1).
 
+.PP
+.TP
+.BI -absname
+Show absolute filenames in error message
+.TP
+.BI -I " dir"
```

Comment:
Spurious extra space?

Reply:
No. These are macro arguments and without it would come out at `-Idir`. Likewise below. The alternative is to use other macros for for changes but they create more line noise.

Reply:
Eep. This is what I get for forgetting troff. No problem then.